cmd/juju/machine/upgradeseries.go

```diff
@@ -28,15 +29,21 @@ const (
 	CompleteCommand = "complete"
 )
 
-var UpgradeSeriesConfirmationMsg = `
-WARNING This command will mark machine %q as being upgraded to series %q
-This operation cannot be reverted or canceled once started. The units
-of machine %q will also be upgraded. These units include:
-
+var upgradeSeriesConfirmationMsg = `
+WARNING: This command will mark machine %q as being upgraded to series %q.
+This operation cannot be reverted or canceled once started.
 %s
-
 Continue [y/N]?`[1:]
 
+var upgradeSeriesAffectedMsg = `
+Units running on the machine will also be upgraded. These units include:
+  %s
```

Comment:
Since the demo, I at least wonder if we want to list both the units and the applications. I don't have a great wording to indicate the leadership of applications involved with affected units will be held.

Maybe check with Rick for a wording

Reply:
It appears like this when there are units:
```
WARNING: This command will mark machine "0" as being upgraded to series "xenial".
This operation cannot be reverted or canceled once started.
Units running on the machine will also be upgraded. These units include:
  redis/0
  wordpress/0

Leadership for the following applications will be pinned and not
subject to change until the "complete" command is run:
  redis
  wordpress

Continue [y/N]?
```